src/rngs/os.rs

```diff
@@ -7,20 +7,29 @@
 // except according to those terms.
 
 //! Interface to the random number generator of the operating system.
+// Note: keep this code in sync with `rand_os` crate!
 
 use getrandom::getrandom;
-use rand_core::{CryptoRng, RngCore, Error, ErrorKind, impls};
+use rand_core::{CryptoRng, RngCore, Error, impls};
 
 /// A random number generator that retrieves randomness from from the
 /// operating system.
 ///
 /// This is a zero-sized struct. It can be freely constructed with `OsRng`.
-/// 
+///
 /// The implementation is provided by the [getrandom] crate. Refer to
 /// [getrandom] documentation for details.
 ///
-/// ## Example
+/// # Usage example
+/// ```
+/// use rand::{RngCore, rngs::OsRng};
+///
+/// let mut key = [0u8; 16];
+/// OsRng.fill_bytes(&mut key);
+/// let random_u64 = OsRng.next_u64();
+/// ```
 ///
+/// PRNG initialization:
```

Comment:
Is it worth having two separate examples? The one below does demonstrate direct usage of `next_u32`.

Reply:
Well, I wanted to remove the second example to keep this file more in sync with `rand_os`, but I thought you added reseeding example for a reason. So if you don't want to keep it I will remove it.

Reply:
It is one of the major use-cases for `OsRng`, hence I showed both.

Reply:
In the example below you demonstrate `u32` generation using `StdRng`, not `OsRng`.

Reply:
Both, actually. Maybe it's not clear enough.